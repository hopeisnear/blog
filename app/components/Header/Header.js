import React from 'react';

import './Header.scss';

export default function Header() {
  return (
    <header className="Header">
      <div className="header__top">
        <div className="header__logo">
          <a href="#">&lt; GOOD Developer &gt;</a>
        </div>
      </div>
    </header>
  );
}
