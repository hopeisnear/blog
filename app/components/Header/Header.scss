.app-header {
  .header-top {
    padding-top: 20px;
    padding-bottom: 20px;
    padding-left: 25px;
    padding-right: 25px;
    display: flex;
    max-width: 1280px;
    margin: 0 auto;
    justify-content: space-between;
  }
  border-bottom: 1px solid rgba(0,0,0,.05);
}


