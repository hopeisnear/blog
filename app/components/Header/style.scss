.app-header {
  .header-top {
    padding-top: 20px;
    padding-bottom: 20px;
    padding-left: 25px;
    padding-right: 25px;
    display: flex;
    max-width: 1280px;
    margin: 0 auto;
    justify-content: space-between;
  }
  .page-menu-nav {
    border-top: 1px solid black;
    background: #fcfcfc;
    .page-menu {
      padding-top: 15px;
      padding-bottom: 15px;
      padding-left: 25px;
      padding-right: 25px;
      display: flex;
      justify-content: space-between;
      max-width: 1280px;
      margin: 0 auto;
    }
  }

}


