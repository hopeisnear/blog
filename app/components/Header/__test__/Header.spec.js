import React from 'react';
import { shallow } from 'enzyme';

import Header from '../Header';

describe('<Header />', () => {
  test('should render <Header />', () => {
    const headerComponent = shallow(<Header />);

    expect(headerComponent).toMatchSnapshot();
  });
});
