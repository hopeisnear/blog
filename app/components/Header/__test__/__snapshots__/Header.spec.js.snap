// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render <Header /> 1`] = `
<header
  className="Header"
>
  <div
    className="header__top"
  >
    <div
      className="header__logo"
    >
      <Link
        replace={false}
        to="/"
      >
        <div>
          &lt; GOOD developer &gt;
        </div>
      </Link>
    </div>
    <div
      className="header__quote"
    >
      <blockquote
        className="quote"
      >
        Adapt what is useful, reject what is useless, and add what is specifically your own.
      </blockquote>
    </div>
  </div>
</header>
`;
